site/js/main.js

```diff
@@ -57,9 +57,6 @@ window.addEventListener('DOMContentLoaded', async () => {
     offlineSearch.style.display = 'block';
   }
 
-  const { Theme } = await import('./components/theme');
-  new Theme();
```

Comment:
Why are you removing this from `main.js`? We need this to load phosphor on site load

Reply:
Turns out it was being called twice - `ee.js` calls a new Theme within its constructor, making this one redundant.